molecule/source-galaxy/group_vars/all

```diff
@@ -2,21 +2,21 @@
 pulp_default_admin_password: password
 pulp_source_dir: '/var/lib/pulp/devel/pulpcore'
 pulp_git_url: "https://github.com/pulp/pulpcore"
-pulp_git_revision: "PULPCORE_PLACEHOLDER"
+pulp_git_revision: "3.20.0"
 pulp_pip_editable: true
 pulp_install_plugins:
   galaxy-importer:
     source_dir: "/var/lib/pulp/devel/galaxy-importer"
     git_url: "https://github.com/ansible/galaxy-importer"
-    git_revision: "IMPORTER_PLACEHOLDER"
+    git_revision: "v0.4.5"
   pulp-ansible:
     source_dir: "/var/lib/pulp/devel/pulp_ansible"
     git_url: "https://github.com/pulp/pulp_ansible"
-    git_revision: "ANSIBLE_PLACEHOLDER"
+    git_revision: "0.14.0"
   pulp-container:
     source_dir: "/var/lib/pulp/devel/pulp_container"
     git_url: "https://github.com/pulp/pulp_container"
-    git_revision: "CONTAINER_PLACEHOLDER"
+    git_revision: "2.13.2"
   galaxy-ng:
```

Comment:
CI uses `sed` to replace these values:
https://github.com/pulp/pulp_installer/blob/main/.github/workflows/galaxy_ci.yml#L58-L72

Reply:
My bad, I was testing it locally this way.